runhouse/resources/hardware/cluster.py

```diff
@@ -739,18 +739,45 @@ def connect_server_client(self, force_reconnect=False):
                 system=self,
             )
 
-    def status(self, resource_address: str = None):
-        """Loads the status of the Runhouse daemon running on the cluster."""
+    def status(self, resource_address: str = None, send_to_den: bool = False):
+        """
+        Loads the status of the Runhouse daemon running on the cluster.
+        :param resource_address: resource's rns address (/<owner_username>/<resource_name>)
```

Comment:
can we use google doc strings in line with the rest of the repo? (same with some of the other methods in `main.py` actually)